components/alternate-accessions.js

```diff
@@ -0,0 +1,26 @@
+// node_modules
+import PropTypes from "prop-types";
+
+/**
+ * // Display the alternate accessions, normally below the header line in objects.
+ */
+export default function AlternateAccession({ alternateAccessions = [] }) {
```

Comment:
Don’t need the double slash comment; it’s already in a standard comment. Rename this to `AlternateAccessions` since that’s the file name, and it does display potentially more than one.